### PR TITLE
Fix in Number Model For Existing Cases

### PR DESCRIPTION
**Pull request by abhimkdm**
Input : مائتان و اثنان ألف

RegEx : RoundNumberIntegerRegex

Added: مائتان و

**Latest Count For Number Model**

  | ARABIC | NumberModel | ExtractorPass | 380
-- | -- | -- | -- | --
  | ARABIC | NumberModel | FullPass | 160
  | ARABIC | NumberModel | Skipped | 15

- [x]  Code is written per the requirement?
- [x]  No duplicate codes?
- [x]  No build error?
- [x]  Ran all test cases?
- [x]  All cases getting passed?
- [x]  Followed reusability?
- [x]  Followed the naming conventions?
- [x]  New change does not negatively impact on performance?
- [x]  Is the code readable?
- [x]  Following the best practices?

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Definitions.Common/Arabic/NumbersDefinitions.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static class NumbersDefinitions
       public const string LangMarker = @"Ara";
       public const bool CompoundNumberLanguage = false;
       public const bool MultiDecimalSeparatorCulture = true;
-      public const string RoundNumberIntegerRegex = @"(?:مائتان|مائة|مائة|مائتين|ثلاثمائه|أربعة مئة|خمسمائة|ستمائة|سبعمائة|ثمان مائة|تسعمائة|تريليون|ترليون|آلاف|تريليونين|تريليونات|مليار|ملياري|مليارات|مليون|مليونان|ملايين|ملايين|ألف|مليونين|ألفين|مئة|الف|ومائتين|المائة|ثلاثمائة|(?<!((مائة)\s(\s)?))مئتان|الفين|بألفين|الآف)";
+      public const string RoundNumberIntegerRegex = @"(?:مائتان|مائة|مائة|مائتين|ثلاثمائه|أربعة مئة|خمسمائة|ستمائة|سبعمائة|ثمان مائة|تسعمائة|تريليون|ترليون|آلاف|تريليونين|تريليونات|مليار|ملياري|مليارات|مليون|مليونان|ملايين|ملايين|ألف|مليونين|ألفين|مئة|الف|ومائتين|المائة|مائتان و|ثلاثمائة|(?<!((مائة)\s(\s)?))مئتان|الفين|بألفين|الآف)";
```
Comment (neudurgeshp) [inline .NET/Microsoft.Recognizers.Definitions.Common/Arabic/NumbersDefinitions.cs:27]: Why added مائتان و ?
مائتان و is not valid round number. Fixed that.
Thanks!

Comment (abhimkdm) [reply .NET/Microsoft.Recognizers.Definitions.Common/Arabic/NumbersDefinitions.cs:27]: Okay, Let me Update the proper fix for this Soon,

**Comment by abhimkdm** [comment]
Fixed below input as well.
مائة و اثنان و ثلاثون خمس

**Latest Count For Number Model**

  | ARABIC | NumberModel | ExtractorPass | 381
-- | -- | -- | -- | --
  | ARABIC | NumberModel | FullPass | 160
  | ARABIC | NumberModel | Skipped | 14


- [x]  Code is written per the requirement?
- [x]  No duplicate codes?
- [x]  No build error?
- [x]  Ran all test cases?
- [x]  All cases getting passed?
- [x]  Followed reusability?
- [x]  Followed the naming conventions?
- [x]  New change does not negatively impact on performance?
- [x]  Is the code readable?
- [x]  Following the best practices?

File: .NET/Microsoft.Recognizers.Definitions.Common/Arabic/NumbersDefinitions.cs (hunk 1 above)
Comment (abhimkdm) [reply .NET/Microsoft.Recognizers.Definitions.Common/Arabic/NumbersDefinitions.cs]: Fixed in ZeroToNineIntegerRegex

**Pull request merged**